src/reconcile/rec_write.c

```diff
@@ -2485,6 +2481,20 @@ __rec_pack_delta_leaf(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_SAVE_UPD *su
 
     WT_ERR(__rec_delta_pack_key(session, S2BT(session), r, supd->ins, supd->rip, key));
 
+    /* 
+     * The max length of a delta:
+     * 1 header byte
+     * 4 timestamps (4 * 9)
+     * key size (5)
+     * key
+     * value size (5)
+     * value
```

Comment:
Thank you for this comment!  I think if you indent these lines, then clang_format will leave them alone.